### ISSUE TITLE
How to set multiple learning rates in different intervals of epoches

### ISSUE THREAD

**Issue by clw5180** [fizyr/keras-retinanet#871]
I know the default value of learning rate is 1e-5 ( isn't that too small? ) in the train.py, but I don't know if I can set multiple learning rates in different intervals of epoches, for example, for epoch 1-10 set lr=0.01, and for epoch 11-20 set lr=0.001..... Is it dynamic or hard-coded?

**Comment by hgaiser** [comment]
Currently you set one learning rate to start with, and a special callback checks if the loss hasn't decreased for a few epochs. If that is the case, the learning rate is reduced by a factor 10.

If you want the learning rate schedule you propose, using the epoch as input, you'll have to make your own learning rate scheduler.

**Comment by pratibhashinde** [comment]
How to make a new learning rate scheduler ?Can anyone help me?
